### ISSUE TITLE
Parceler compatibility?

### ISSUE THREAD

**Issue by xdgimf** [workarounds/bundler#8]
I'm not sure if the library supports parceler objects. I tried it and its not working. Any tips to make this work?
````
@RequireBundler
public class StatsActivity extends AppCompatActivity {

    @Arg
    EventsItem mItem; // Parcel Object

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Bundler.inject(this); // Not generated;
    }

}
````

**Comment by madki** [comment]
Parceler object is not currently supported. Currently working on adding support. Will update once done.

**Comment by madki** [comment]
Fixed in 4c26f16b2911f987fbd8b00077069cfb9727d62a. Parceler is now compatible.

Use `v0.1.1` or later. Add the additional dependency
```groovy
  compile "in.workarounds.bundler:bundler-parceler:$bundlerVersion"
```

Then modify the activity above as:
```java
@RequireBundler
public class StatsActivity extends AppCompatActivity {
  @Arg(serializer = ParcelerSerializer.class)
  EventsItem mItem; // Parcel object

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    Bundler.inject(this); // will work now.
  }
}
```
